modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -3607,64 +3571,143 @@ enum GmosNorthGrating {
-  # Explicitly specified GMOS Y-Binning, override the default (TWO)
+
+  # GMOS Y-Binning, either explicitly specified in explicitYBin or else taken
+  # from the defaultYBin.
+  yBin: GmosYBinning!
+
+  # Default GMOS Y-Binning (TWO).
+  defaultYBin: GmosYBinning!
+
+  # Optional explicitly specified GMOS Y-Binning. If set it overrides the
+  # default.
   explicitYBin: GmosYBinning
 
-  # Explicitly specified GMOS amp read mode, override the default (SLOW)
+
+  # GMOS amp read mode, either explicitly specified in explicitAmpReadMode or
+  # else taken from the defaultAmpReadMode.
+  ampReadMode: GmosAmpReadMode!
+
+  # Default GmosAmpReadMode (SLOW).
+  defaultAmpReadMode: GmosAmpReadMode!
+
+  # Optional explicitly specified GMOS amp read mode. If set it overrides the
+  # default.
   explicitAmpReadMode: GmosAmpReadMode
 
-  # Explicitly specified GMOS amp gain, override the default (LOW)
+
+  # GMOS amp read gain, either explicitly specified in explicitAmpGain or else
+  # taken from the defaultAmpGain.
+  ampGain: GmosAmpGain!
+
+  # Default GMOS amp gain (LOW).
+  defaultAmpGain: GmosAmpGain!
+
+  # Optional explicitly specified GMOS amp gain.  If set it override the default.
   explicitAmpGain: GmosAmpGain
 
-  # Explicitly specified GMOS ROI, overriding the default (FULL_FRAME)
+
+  # GMOS ROI, either explicitly specified in explicitRoi or else taken from the
+  # defaultRoi.
+  roi: GmosRoi!
+
+  # Default GMOS ROI (FULL_FRAME).
+  defaultRoi: GmosRoi!
+
+  # Optional explicitly specified GMOS ROI. If set it overrides the default.
   explicitRoi: GmosRoi
 
-  # Explicitly specified wavelength dithers required to fill in the chip gaps (in nm), taking the place of the calculated value based on the grating dispersion
+
+  # Wavelength dithers required to fill in the chip gaps, expressed in nanometers.
+  # This value is either explicitly specified in explicitWavelengthDithersNm
+  wavelengthDithersNm: [BigDecimal!]!
+
+  # Default wavelength dithers, calculated based on the grating dispersion.
+  defaultWavelengthDithersNm: [BigDecimal!]!
+
+  # Optional explicitly specified wavelength dithers.  If set it overrides the
+  # default.
   explicitWavelengthDithersNm: [BigDecimal!]
 
-  # Explicitly specified spatial q offsets, overriding the default
+
+  # Spacial q offsets, either explicitly specified in explicitSpatialOffsets
+  # or else taken from defaultSpatialOffsets
+  spatialOffsets: [q!]!
+
+  # Default spatial offsets.
+  defaultSpatialOffsets: [q!]!
+
+  # Optional explicitly specified spatial q offsets. If set it overrides the
+  # the default.
   explicitSpatialOffsets: [q!]
+
 }
 
-# GMOS North Long Slit basic configuration
-type GmosNorthLongSlitBasic {
+# GMOS North Long Slit mode
+type GmosNorthLongSlit implements GmosLongSlitObservingMode {
+
   # GMOS North Grating
   grating: GmosNorthGrating!
 
   # GMOS North Filter
   filter: GmosNorthFilter
 
   # GMOS North FPU
-  fpu: GmosNorthBuiltinFpu!
+  fpu: GmosNorthBuiltinFpu
```

Comment:
`grating`, `filter` and `fpu` are set when the observing mode is selected from the table but can be subsequently updated.